docs/specs/xref.yml

```diff
@@ -45,6 +45,56 @@ outputs:
   docs/a.json: |
     { "conceptual": "<p>Link to <a class=\"no-loc\" href=\"/dotnet/api/system.string\">System.String</a></p>\n"}
 ---
+# Remove external xref url hostname 
+# when config hostname is old one but xref url hostname is matched new one
+inputs:
+  docfx.yml: |
+    hostName: docs.microsoft.com
+    hostNameMapping: 
+      docs.microsoft.com: new-docs.microsoft.com
+    extend: _shared/extend.json
+  _shared/extend.json: |
+    {"xref":["1.xrefmap.json"]}
+  docs/a.md: Link to @System.String?displayProperty=fullName
+  1.xrefmap.json: |
+    {
+      "references":[{
+          "uid": "System.String",
+          "name": "String",
+          "fullName": "System.String",
+          "href": "https://new-docs.microsoft.com/dotnet/api/system.string",
```

Comment:
I think here we still need to output `docs.microsoft.com`

Reply:
@herohua   `1.xrefmap.json` is an input configuration.
support reading external xrefmap.json files is a feature provided by DocFX